pdctl/command/region_command.go

```diff
@@ -42,7 +43,7 @@ type regionInfo struct {
 // NewRegionCommand return a region subcommand of rootCmd
 func NewRegionCommand() *cobra.Command {
 	r := &cobra.Command{
-		Use:   "region <region_id>",
+		Use:   `region <region_id> [-jq="<query string>"]`,
```

Comment:
only support some commands?

Reply:
Yes. Only support APIs that query region/store list. The results returned by other APIs are simple and easy to see with the naked eye.